File: homework10/Q7/q7.js
//7. Write the following programs:
 //- Write a JavaScript program that will create another array, which will be 2/3 * the corresponding 
//element in the new array + 1/2. Then, for each element in b, print element * every element in the new array 
	//(cross product). .


var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
var c = new Array(b.length);      

{
    var i;
for( i=0 ;i<c.length; i++){
c[i]=b[i]*(2/3)+(1/2);
console.log(c[i]);
}
console.log('______________');
for( i=0 ;i<c.length; i++){
var sum=0;
var j;
for( j=0 ;j<c.length; j++)
sum=sum+b[i]*c[j];
console.log(sum);
}

}

File: homework10/Q7/q7.rb
# 7. Write the following programs:
# - Write a JavaScript program that will create another array, which will be 2/3 * the corresponding 
#  element in the new array + 1/2. Then, for each element in b, print element * every element in the new array 
#  (cross product).
 
 
 b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
 c = Array.new(b.length);      


i=0
while i<c.length
c[i]=b[i]*(0.667)+(0.5)
puts (c[i])
i=i+1
end

puts ('______________');


sum=0;
i=0
while i<c.length
sum=sum+c[i]
i=i+1
end
j=0 
while j<b.length
puts (sum*b[j])
j=j+1
end
